### DISCUSSION TITLE
does Harman sdk collect personalized information about the users?

### DISCUSSION THREAD

**Discussion by ronyarbiv** [airsdk/Adobe-Runtime-Support#909] [Q&A] [answered]
Hi,

does Harman sdk collect personalized information about the users?

**Comment by marchbold** [comment]
Probably need to clarify your question here, are you referring to usage of the AIR SDK to build an application or once you've built an app? 

I believe there is some stats gathered around the usage of `adt` to package an application, but I doubt there's anything once an application has been built.

**Reply by ronyarbiv** [reply]
I have sent an empty app including only air sdk 33.1.47 for review and got rejected on google play store.

**Reply by marchbold** [reply]
There's definitely nothing in an empty app that should get you rejected from review.

**Comment by ajwfrost** [comment, marked as answer]
@marchbold is right: we do some tracking to see what platforms and targets are used in ADT, but there is no tracking of end users or any callbacks or similar once you’ve packaged up an application. Adobe did have code in there for anonymous installation metrics but they took that out prior to GDPR laws coming in. So from an end user perspective - we don’t have anything that would cause anyone a concern here…

**Reply by ajwfrost** [reply]
Also just from https://airsdk.harman.com/assets/pdfs/HARMAN%20AIR%20SDK%20License%20Agreement.pdf, section 6.4 on Privacy:

> The Runtime embedded within such Developer Applications will not send your end user information back to HARMAN.
> The SDK and Build tools will send your information back to HARMAN in order to confirm your 
> subscription to or payment of licensing fees, as well as to help HARMAN improve its products and 
> services. Examples of some of the information that will be sent to HARMAN include the following: (a)
> operating system; (b) application ID; (c) packaging settings; (d) authoring tool; (e) SDK version; 
> and/or (f) anonymised identifier unique to each machine on which you use the SDK.

**Comment by ronyarbiv** [comment]
Thank you.
